site/viamguides/src/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,223 @@
+  ![Code in VS Code](assets/vscode.png)
+1. **Run the code:** Run the sample code to connect to your machine
+   ```bash
+   python3 square.py
+   ```
+
+If you want to play with the rover, go to the Viam app **CONTROL** tab to interact with your machine’s resources. Click on one of the camera panels and toggle the camera stream on so you can observe the rover’s movements. Or try adding more code to [drive the rover in a square](https://docs.viam.com/get-started/drive-rover/).
+
+In the next few sections, learn how to use variables on your machine in production.
+
+<!-- ------------------------ -->
+## Add the control code to your machine
+Duration: 3
+
+Once you have the code working in your local development environment, you may choose to move your control code to your machine, so that it can run in production. 
+
+> aside positive
+> If you followed [Try Viam](https://app.viam.com/try), you won't have access to add control code to the machine, since you are just borrowing the rover. However, you can skip to the next part to see how you can set up variables on the machine. If you're using your own wheeled robot, you can complete these steps.
+
+### Add the control code to your machine
+1. **SSH into your board**: From the terminal window, run the following command to [SSH (Secure Shell) into your board](https://docs.viam.com/installation/prepare/rpi-setup/#connect-with-ssh), where the text in `<>` should be replaced (including the `<` and `>` symbols themselves) with the `user` and `hostname` you configured when you set up your machine. 
+    ```bash
+    ssh <USERNAME>@<REMOTE-HOSTNAME>.local
+    ```
+1. **Create a project directory**: Create a new directory on your machine to hold your project code.
+    ```bash
+    mkdir robot
+    ```
+1. **Install dependencies**: Install the Viam Python SDK (and other dependencies if required) into the folder. 
+    ```bash
+    pip3 install viam-sdk
+    ```
+<!-- TODO do i need to activate a venv? -->
+<!-- TODO do i need to chmod? -->
+1. **Copy file(s) to the machine**: From the original computer you were developing on (not the secure shell prompt accessing your machine), copy the `square.py` file to your machine.
+    ```bash
+    scp square.py <USERNAME>@<REMOTE-HOSTNAME>.local:/home/myboard/robot/square.py
+    ```
+1. **Add a process**: [Configure a process](https://docs.viam.com/configure/processes/#configure-a-process) by going to the **CONFIGURE** tab of the Viam app. Click the plus icon (**+**) next to your machine in the left-hand menu and select **Process**. Name your process `process-square`.
+  ![Add a process](assets/process.png)
+1. **Find executable path**: To find out the path of the Python 3 executable that is being used on your machine, you can input `which python3` in the command line from within your secure shell. You will need this output for the upcoming steps.
```

Comment:
`which python3` should be run after activating the virtual environment under which the viam-sdk and other dependencies were installed.